### PR TITLE
ofnet forwarding mode changes

### PR DESCRIPTION
**Pull request by abhinandanpb**
Add Cleanup to allow forwarding mode to be changed by deleting and recreating ovs switch instances.

### CODE REVIEW

File: ofnetPolicy_test.go [hunk 1]
```diff
@@ -221,5 +221,5 @@ func TestPolicyAddDelete(t *testing.T) {
 	log.Infof("Verified all flows are deleted")
 
 	// Delete the bridge
-	ovsDriver.DeleteBridge(brName)
+	//ovsDriver.DeleteBridge(brName)
```
Comment (shaleman) [inline ofnetPolicy_test.go]: Lets remove this line if its not needed

File: ofnetSvcProxy_test.go [hunk 2]
```diff
@@ -391,5 +391,5 @@ func TestSvcProxyInterface(t *testing.T) {
 	verifyWatchRemoved(t, brName)
 
 	// Delete the bridge
-	ovsDriver.DeleteBridge(brName)
+	//	ovsDriver.DeleteBridge(brName)
```
Comment (shaleman) [inline ofnetSvcProxy_test.go]: same here

File: rpcHub/rpcHub.go [hunk 3]
```diff
@@ -157,3 +165,54 @@ func (self *RpcClient) Call(serviceMethod string, args interface{}, reply interf
 
 	return err
 }
+
```
Comment (shaleman) [inline rpcHub/rpcHub.go]: Is this code being replicated here from netutils just to avoid dependency on netplugin?

**Comment by shaleman** [comment]
LGTM other than minor comments.
This needs to be rebased I think. Sanity failure should also go away if you rebase..

File: rpcHub/rpcHub.go (hunk 3 above)
Comment (abhinandanpb) [inline rpcHub/rpcHub.go]: yep . I hit circular dependency issue.

File: ofnetPolicy_test.go (hunk 1 above)
Comment (abhinandanpb) [inline ofnetPolicy_test.go]: sure will get rid of it

**Pull request merged**